app/helpers/tweet_helpers.rb

```diff
@@ -0,0 +1,5 @@
+module TweetHelpers
+  def validate_new_tweet_params(params)
+    params[:content] != ""
```

Comment:
I think you could use [ActiveRecord validations](https://learn.co/tracks/full-stack-web-development/sinatra/activerecord/user-authentication-in-sinatra) (e.g. `validates_presence_of :username`) to perform these types of checks more easily.